frontend/src/styles/components/alert.css

```
@@ -0,0 +1,30 @@
+.alert {
+  --border-color: transparent;
+  align-items: center;
```

For consistency purposes 💐 

```suggestion
.alert {
  --border-color: transparent;

  align-items: center;
```